lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1588,7 +1555,7 @@ input[name="subject"] {
       font-size: 12px;
       color: #646464
     }
-    top:-95px !important;
+    top: 100px !important;
```

Comment:
modal dialog was rendering way to high on the page. Not sure why the change of previous behavior.